tests/conftest.py

```diff
@@ -270,12 +269,6 @@ def audit_board_round_2_ids(
     yield [ab["id"] for ab in audit_boards]
 
 
-@pytest.fixture
-def as_jurisdiction_admin(client: FlaskClient, jurisdiction_ids: List[str]):
-    create_jurisdiction_admin(jurisdiction_ids[0])
-    set_logged_in_user(client, UserType.JURISDICTION_ADMIN, DEFAULT_JA_EMAIL)
```

Comment:
this didn't work so well because there's no guaranteed order to the fixtures getting called